includes/class-wcmp-data.php

```diff
@@ -58,6 +58,7 @@ class WCMP_Data
 
     public const DEFAULT_COUNTRY_CODE = "NL";
     public const DEFAULT_CARRIER       = CarrierPostNL::ID;
+    public const DEFAULT_CARRIER_NAME  = CarrierPostNL::NAME;
```

Comment:
`(self::DEFAULT_CARRIER_CLASS)::ID`
`(self::DEFAULT_CARRIER_CLASS)::NAME`

of daar gewoon zo schrijven. Ik vind het ook niet heel mooi zo, maar het punt van alleen de `CLASS` hebben is dat je daarmee ook bij die consts kan komen